src/cpp/core/system/Crypto.cpp

```diff
@@ -535,9 +547,21 @@ core::Error rsaPrivateDecrypt(const std::string& cipherText, std::string* pPlain
    status = EVP_PKEY_decrypt_init(ctx.get());
    if (status <= 0)
       return getLastCryptoError(ERROR_LOCATION);
+
+   status = EVP_PKEY_CTX_set_rsa_oaep_md(ctx.get(), EVP_sha256());
+   if (status <= 0)
+      return getLastCryptoError(ERROR_LOCATION);
+   
+   status = EVP_PKEY_CTX_set_rsa_padding(ctx.get(), RSA_PKCS1_OAEP_PADDING);
+   if (status <= 0)
+      return getLastCryptoError(ERROR_LOCATION);
```

Comment:
Needed, since we're now using a SHA-256 digest function, as well as OAEP padding, when encrypting things on the client side.